src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7327,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <value>Visualize Your Fundraising Processes</value>
+    </labels>
+      <labels>
+        <fullName>gseuChecklistWorkRecordsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Work with Records checklist in Get Started for End Users</shortDescription>
+        <value>After you get your constituent and donation data into NPSP, you need to be able to easily find and interact with the records you&apos;re working on.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistWorkRecordsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Work with Records checklist in Get Started for End Users</shortDescription>
+        <value>Work with Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Global Search checklist item in Get Started for End Users</shortDescription>
+        <value>When you&apos;re just getting started with NPSP, finding records can be tricky. Use global search to search across your entire Salesforce org and find what you need fast.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Global Search in GS for End Users</shortDescription>
+        <value>Learn more about Search Results in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of Global Search checklist item in Get Started for End Users</shortDescription>
+        <value>Find Records</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemGlobalSearchTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Global Search checklist item in Get Started for End Users</shortDescription>
+        <value>Find Records with Global Search</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStayTrackDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Stay Track in GS for End Users</shortDescription>
```

Comment:
Please change this to: Description of Stay on Track checklist item in Get Started for End Users